Source/engine/point.hpp

```diff
@@ -10,6 +12,30 @@ struct Point {
 	int x;
 	int y;
 
+	static constexpr Point fromDirection(Direction direction)

```

Comment:
Can't this then replace the the identical switch in `+=(Direction direction)` on line 58?

Reply:
Yes, that was part of the original PR but I didn't catch all the dropped changes after the rebase. I'll update it tomorrow, getting late now where I am.